scripts/integration-tests/utils/e2e-check.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-process-exit */
+import fs from "fs";
+import github from "@actions/github";
+
+const { repo } = github.context;
+
+const resp = await fetch(
+  "https://raw.githubusercontent.com/babel/babel/main/package.json"
+);
+const mainRepoPackageJson = await resp.json();
+
+if (
+  mainRepoPackageJson.version !==
+  JSON.parse(fs.readFileSync("package.json")).version
+) {
+  if (repo.owner === "babel" && repo.repo === "babel") {
```

Comment:
```suggestion
  if (process.env.GITHUB_REPOSITORY === "babel/babel") {
```
no need to pull in `@actions/github`